runtime/src/bank.rs

```diff
@@ -715,10 +715,10 @@ impl Bank {
 
     /// Collect the node Pubkey and staker account balance for nodes
     /// that have non-zero balance in their corresponding staker accounts
-    pub fn staked_nodes(&self) -> HashMap<Pubkey, u64> {
+    pub fn node_stakes(&self) -> HashMap<Pubkey, u64> {
         self.vote_states(|state| self.get_balance(&state.staker_id) > 0)
             .iter()
-            .map(|state| (state.node_id, self.get_balance(&state.staker_id)))
+            .map(|(vote_account, state)| (state.delegate_id, self.get_balance(vote_account)))
```

Comment:
Looks like we need a test for multiple stakers delegating to the same `delegate_id`.

Reply:
Yup that bug still exists. I'll get to it tomorrow.